public/scripts/frames.js

```diff
@@ -117,6 +118,7 @@ function setDataIntoCard() {
             tickets += frame;
 
             document.getElementById("cardsList").innerHTML = tickets;
+            fillLogWork(title);
```

Comment:
Let's not call this from here let's call this from the button click of log, joel can do this, as he having button.

And fillLogWork method will not have title, rather it will have the list of all the arguments that you need to fill in the ui

Reply:
ok